repository-management.md

```diff
@@ -0,0 +1,101 @@
+# Repository Management
+
+## Package Names
+
+Package names for the official `wp-cli` GitHub repository need to be discussed upfront with the [maintainers](https://github.com/orgs/wp-cli/teams/maintainers).
+
+Package names for WP-CLI commands must end with the `-command` suffix.
+
+_Example: `wp-cli/scaffold-package-command`._
+
+## Package Versions
+
+Package version follow the [Semantic Versioning Specification (SemVer)](http://semver.org/).
+
+## Milestones
+
+Open milestones usually point to the next version to be released.
+
+As the release notes are built using the milestones, pull requests that are to be merged need to be set to the upcoming milestone, so that these pull requests will be part of the release notes.
+
+## Labels
+
+Labels are an important organizational tool to communicate the state and progress of the work being done. [Committers](https://github.com/orgs/wp-cli/teams/committers) are required to keep these up-to-date.
+
+### Groups Of Labels
+
+Labels can be part of a label group, a concept that is applicable to all packages. The actual labels that are available can depend on the actual package, but the groups always have the same semantic role.
+
+#### Command
+
+The labels that define what exact command a given issue/pull request applies to are prefixed with `command:`
+
+_Example: `command:cli-update`_
+
+#### Scope
+
+The labels that define what scope the current issue/pull request applies to are prefixed with `scope:`.
+
+Used scopes:
+
+* `bootstrap` - Part of the bootstrap process, which loads both WP-CLI as well as WordPress Core.
```

Comment:
For clarity, I'd use `scope:bootstrap` instead of `bootstrap` when referring to specific labels. Doing so the reader doesn't have to connect the docs in their head.

Reply:
https://github.com/wp-cli/handbook/pull/127/commits/95f0f62007eea934fc4f47269aae81a4107104be